src/exercises/geojson/points/08.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from 'react';
+
+import { ViewState } from 'react-map-gl';
+
+import { Story } from '@storybook/react/types-6-0';
+import { FeatureCollection, Point } from 'geojson';
+
+import Map from 'components/map';
+import Controls from 'components/map/controls';
+import ZoomControl from 'components/map/controls/zoom';
+import ClusterLayer from 'components/map/layers/clusters';
+import MarkerCluster from 'components/map/markers/cluster';
+import MarkerPin from 'components/map/markers/pin';
+import { CustomMapProps } from 'components/map/types';
+import AIRPORTS_DATA from 'data/airports.json';
+
+export type AirportDataProperties = {
+  scalerank: number;
+  featurecla: string;
+  type: string;
+  name: string;
+  abbrev?: string;
+  location: string;
+  gps_code?: string;
+  iata_code?: string;
+  wikipedia?: string;
+  natlscale: number;
+  cartodb_id: number;
+  created_at: string;
+  updated_at?: string;
+};
+
+const StoryMap = {
+  title: 'Exercises/Geojson/Points',
+  component: Map,
+  argTypes: {},
+};
+
+export default StoryMap;
+
+const Template: Story<CustomMapProps> = (args: CustomMapProps) => {
+  const { id, bounds, initialViewState } = args;
+
+  const [viewState, setViewState] = useState<Partial<ViewState>>();
+
+  const onSelectMarker = (props: AirportDataProperties) => {
+    //  do something with marker
```

Comment:
Could you use `getClusterExpansionZoom` from supercluster and move to next expansion zoom?